tests/integration.rs

```diff
@@ -3489,24 +3489,51 @@ mod common_parallel {
         _test_virtio_block(FOCAL_IMAGE_NAME, true, true);
     }
 
+    fn disk_check_consistency_path(path: &str) {
+        let output = std::process::Command::new("qemu-img")
+            .args(["check", path])
+            .output()
+            .expect("Expect checking disk image consistency");
+
+        assert!(
+            output.status.success(),
+            "qemu-img check failed: {}",
+            String::from_utf8_lossy(&output.stderr)
+        );
+    }
+
+    fn disk_check_consistency(image_name: &str) {
```

Comment:
```suggestion
    fn disk_check_consistency(image_name: impl AsRef<Path>) {
```

And please remove `disk_check_consistency_path` then and directly embed it into `disk_check_consistency`.

Reply:
ah, I see. Sorry, I missed that. Nevertheless, I think we can handle this with one single function consuming `impl AsRef<Path>` which is more idiomatic. You then can either directly run the check if `path.exists()` or do your magic to prepend the workloads directory to the path before performing the check.

I think this will result in more idiomatic code

Reply:
Okay. I don't mind changing the code.